packages/sync/src/links/OfflineQueueLink.ts

```diff
@@ -67,6 +70,9 @@ export class OfflineQueueLink extends ApolloLink {
       forward(operation).subscribe(observer);
     });
     this.opQueue = [];
+    if (this.listener && this.listener.queueCleared) {
```

Comment:
Sorry for the idiot question but what `this.listener.queueCleared` do in the if/test?

Reply:
Both listener and individual methods are optional and can be undefined